PhysicsTools/PatAlgos/plugins/PATTauProducer.cc

```diff
@@ -397,9 +398,11 @@ void PATTauProducer::produce(edm::Event & iEvent, const edm::EventSetup & iSetup
       aTauPFEssential.etaAtEcalEntranceLeadChargedCand_ = leadChargedCandEtaAtEcalEntrance;
       if (sumEnergy != 0.) {
         aTauPFEssential.etaAtEcalEntrance_ = sumEtaTimesEnergy/sumEnergy;
+	aTauPFEssential.phiAtEcalEntrance_ = sumPhiTimesEnergy/sumEnergy;
```

Comment:
You can leave this stuff in but I would not add it to the next PR. We take this one on a separate note.